### PR TITLE
{Network} Bump azure-mgmt-network from 20.0.0 to 21.0.1

### PR DESCRIPTION
**Pull request by necusjz**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Network

**Review by kairu-ms — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Review by calvinhzy — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py [hunk 1]
```diff
@@ -1184,6 +1184,7 @@ def test_postgres_flexible_server_vnet_mgmt_supplied_vname_and_subnetname(self,
     def test_mysql_flexible_server_vnet_mgmt_supplied_vname_and_subnetname(self, resource_group):
         self._test_flexible_server_vnet_mgmt_supplied_vname_and_subnetname('mysql', resource_group)
 
+    @live_only()
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py:1187]: May I know why this test need to be live only? Shall I involve RDBMS team contributor to fix the test?

Comment (necusjz) [reply src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py:1187]: I already contacted them via Teams and @alanenriqueo will work on that.

**Review by evelyn-ys — APPROVED** [review]

**Review by yonzhan — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Comment by jiasli** [comment]
Force merging.

**Pull request merged**